db/migrate/20180112082901_devise_create_admin_users.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+# rubocop:disable Metrics/AbcSize
```

Comment:
__[RuboCop]__

Lint/MissingCopEnableDirective: Re-enable Metrics/AbcSize cop with `# rubocop:enable` after disabling it.


<!-- Comment by [SideCI](https://sideci.com) -->